patches/tModLoader/Terraria/Localization/Content/en-US/tModLoader.json

```diff
@@ -125,6 +122,8 @@
 		"ModVersionInfoUnchanged": "Error: Mod Version information must be incremented prior to updating.",
 		"ConsultSteamLogs": "Consult Steam workshop_log.txt (Default Location: {0}) for more information.",
 		"SteamRejectUpdate": "Item is/was already installed: {0}. \nIf attempting to re-install, restart your game first because Steam is weird\nIf restarting your game does not work, apply the following steps:\n1) Open Steam Client\n2) Find the Mod on tModLoader Workshop\n3) Unsubscribe and Resubscribe on the actual Workshop.\nThese steps are required if Steam fails to handle unsubscribing you properly.",
+		"BetaModCantPublishError": "You cannot publish a Mod built on a beta version of tModLoader",
+		"WrongVersionCantPublishError": "You cannot publish a Mod built on a different version of tModLoader than what it was built with",
```

Comment:
Please don't move these around - I don't think our tracking scripts are designed to handle it.
May screw up other language files.

Deleting unused is OK though